packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor-locked-figures.test.tsx

```diff
@@ -138,56 +130,69 @@ describe("InteractiveGraphEditor locked figures", () => {
             expect(onChangeMock).not.toBeCalled();
         });
 
-        test("Calls onChange when a locked point's x coordinate is changed", async () => {
+        test("Shows settings accordion when a locked $figureType is passed in", async () => {
+            // Arrange
+            renderEditor({
+                lockedFigures: [getDefaultFigureForType(figureType)],
+            });
+
+            // Act
+            // There may be accordions within the settings, such as the
+            // defining point accordions within the line settings.
+            const accordionHeading = screen.getAllByRole("heading", {
+                level: 2,
+            })[0];
+            const capitalizedFigureType =
+                figureType.charAt(0).toUpperCase() + figureType.slice(1);
+
+            // Assert
+            expect(accordionHeading).toBeInTheDocument();
+            expect(accordionHeading).toHaveTextContent(capitalizedFigureType);
```

Comment:
Could we use `figureName` instead of `capitalizedFigureType`?

Reply:
Oh yes! I wrote this test before adding `figureName` to the each. Thank you!